packages/react-dom-bindings/src/client/ReactDOMComponent.js

```diff
@@ -834,6 +833,12 @@ export function setInitialProperties(
       // We listen to this event in case to ensure emulated bubble
       // listeners still fire for the invalid event.
       listenToNonDelegatedEvent('invalid', domElement);
+
+      let type = null;
+      let value = null;
+      let defaultValue = null;
+      let checked = null;
+      let defaultChecked = null;
```

Comment:
you wrote these 5 variables in 3 different orders in the same function

Reply:
You mean in the argument order? The assignments are copy-paste.